todo-app/src/TasksStore.js

```diff
@@ -1,52 +1,136 @@
-import {loadTasks, saveTasks} from './localStorageUtils';
-import {observable, computed, autorun, action} from 'mobx'
+//import {loadTasks, saveTasks} from './localStorageUtils';
+import {observable, computed , action , runInAction} from 'mobx';
+
+const api_url = "https://hyf-react-api.herokuapp.com";
 
 class TasksStore {
 
     @observable
-    tasks = loadTasks();
+    tasks = []
 
-    constructor() {
-        autorun(() => {
-            saveTasks(this.tasks);
-        });
-    }
+    @observable
+    loadingTasks = false 
 
     findTask(taskId) {
         return this.tasks.find(task => task.id === taskId)
     }
 
+    @action
+    loadTasks(showLoader = true){
+        if (showLoader) {
+            this.loadingTasks = true
+        }
+        fetch(`${api_url}/todos`)
+        .then(res =>{return res.json()})
+        .then(tasks_arr => {
+            runInAction(()=>{
+                this.tasks = tasks_arr.map(comment => {
+                    return {
+                        id:comment._id , 
```

Comment:
So, the only property you are actually changing is `id` (from `_id`).

If the object was very large, or if you did not care about retaining the original `_id` property, you could also do:

    return {
      id: comment._id,
      ...comment,
    }